src/engine/renderer/tr_shader.cpp

```diff
@@ -5979,6 +5980,18 @@ static shader_t *FinishShader()
 		ret->depthShader = nullptr;
 	}
 
+	if ( glConfig2.materialSystemAvailable && !tr.worldLoaded ) {
+		uint8_t maxStages = 0;
+		for ( shaderStage_t* pStage = ret->stages; pStage < ret->lastStage; pStage++ ) {
+			maxStages++;
+		}
+
+		if ( maxStages % 4 != 0 ) { // Aligned to 4 components
```

Comment:
You could use the PAD macro

Reply:
Interesting, didn't know we had that macro. Done now.